### ISSUE TITLE
ionic 7 app - install fails

### ISSUE THREAD

**Issue by fitnexus** [twinssbc/Ionic2-Calendar#652]
Getting unresolved dependency issues when trying to install this plugin to Ionic 7

ionic environment details: 
```
 Ionic CLI                     : 7.1.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 7.0.3
   @angular-devkit/build-angular : 15.2.6
   @angular-devkit/schematics    : 15.2.6
   @angular/cli                  : 15.2.6
   @ionic/angular-toolkit        : 9.0.0
```

the error i get: 
```
$ npm install ionic2-calendar  --save    
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: newapp@0.0.1
npm ERR! Found: @ionic/angular@7.0.11
npm ERR! node_modules/@ionic/angular
npm ERR!   @ionic/angular@"^7.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @ionic/angular@"^6.1.9" from ionic2-calendar@1.0.1
npm ERR! node_modules/ionic2-calendar
npm ERR!   ionic2-calendar@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/username/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2023-06-07T14_50_08_519Z-debug-0.log
```

**Comment by twinssbc** [comment]
I have published a new version, 2.0.0, feel free to give a try.
One thing to note is that the I'm still working on Swiper 9 integration. So when installing the Swiper dependency, just use 8.4.6 version.

**Comment by narik1989** [comment]
Hi,

Thanks for the brilliant update for Angular 16 support. Any update on when the Swiper 9 support will be available? Because if i downgrade the existing application from swiper 9 to 8.4.6 i am facing many issues in my existing application.

It would be great to know if there is any update on the same.

**Comment by daviddickson** [comment]
> I have published a new version, 2.0.0, feel free to give a try. One thing to note is that the I'm still working on Swiper 9 integration. So when installing the Swiper dependency, just use 8.4.6 version.

Is the documentation for 8.x.x available anywhere?

**Comment by twinssbc** [comment]
I have published version 2.1.0, which supports swiper 10.1.0

**Comment by twinssbc** [comment]
I have published version 2.1.0, which supports swiper 10.1.0
